### ISSUE TITLE
npm install phantomjs has error message 

### ISSUE THREAD

**Issue by bluemsn** [Medium/phantomjs#134]
CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
  SOLINK_MODULE(target) Release/bufferutil.node
  SOLINK_MODULE(target) Release/bufferutil.node: Finished
  CXX(target) Release/obj.target/validation/src/validation.o
  SOLINK_MODULE(target) Release/validation.node
  SOLINK_MODULE(target) Release/validation.node: Finished
npm ERR! phantomjs@1.9.7-1 install: `node install.js`
npm ERR! Exit status 8
npm ERR!
npm ERR! Failed at the phantomjs@1.9.7-1 install script.
npm ERR! This is most likely a problem with the phantomjs package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node install.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls phantomjs
npm ERR! There is likely additional logging output above.
npm ERR! System Darwin 13.0.2
npm ERR! command "node" "/usr/local/bin/npm" "install"
npm ERR! cwd /Users/zhanglei/workspace/test/mean
npm ERR! node -v v0.10.24
npm ERR! npm -v 1.3.21
npm ERR! code ELIFECYCLE

why ?

**Comment by junibrosas** [comment]
I have the same issue and I resolved it by updating the npm.

**Comment by joaquinalcerro** [comment]
I have Apple OS X - El Capitan and I did the following and looks like it's working:

sudo npm -g install phantomjs2

When I executed this command, If failed installing.... it never finished but the phantomjs v2 gets downloaded to this directory

/usr/local/lib/node_modules/phantomjs2/phantomjs/phantomjs-2.0.0-20141016-macosx.zip-extract-1447034534278/phantomjs-2.0.0-20141016

It will contain a bin and examples directory

Go into the bin directory where you will find the phantomjs file 50MB approximately.

Check in what directory phantomjs file located:

which phantomjs (i got = /usr/local/bin/phantomjs but this file points to: /usr/local/lib/node_modules/phantomjs2/bin/phantomjs)

As root (sudo cp) copy the downloaded phantomjs file to this directory (in my case = /usr/local/lib/node_modules/phantomjs2/bin/). NOTE: the last part, phantomjs is the file name. Don't include it.

Check that this file is executable. If it is not executable, use chmod to make it executable.

Phantomjs should be installed now.

Try

phantomjs --version
